src/en/standards/standard.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Standard on Information and Communication Technology Accessibility (<abbr>ICT</abbr>) Accessibility (Initial)"
+title: "Standard on Information and Communication Technology Accessibility (<abbr>ICT</abbr>) Accessibility (Phase One)"
```

Comment:
In French it is the actual number.

I suggest to be consistent between both language, either use a number or the word or both:

```suggestion
title: "Standard on Information and Communication Technology Accessibility (<abbr>ICT</abbr>) Accessibility (Phase 1)"
```

```suggestion
title: "Standard on Information and Communication Technology Accessibility (<abbr>ICT</abbr>) Accessibility (Phase one (1))"
```

Reply:
Updated French to match English title in new commit.